plugins/rest/plugin.spec.yaml

```
@@ -13,6 +13,9 @@ resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/rest
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
   docs_url: https://docs.rapid7.com/insightconnect/http-requests
+key_features: ["it can do this", "and that"]
+requirements: ["python3", "requests"]
```

Requirements here does not refer to `requirements.txt` but rather what a user needs in order to use the plugin. In this case, they don't need anything technically, other than the details of the API they want to connect to.
So something like this:
```suggestion
requirements: ["None"]
```